src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -72,23 +68,22 @@ public int getSize() {
         return size;
     }
 
-    private int hashKey(K key) {
+    private int getIndex(K key) {
         return key == null ? 0 : Math.abs(key.hashCode() % table.length);
     }
 
     public void resize() {
-        Node<K,V> [] newTable = new Node[table.length * INCREASE_ARRAY_SIZE_INDEX];
-        threshold = (int) (newTable.length * LOAD_FACTOR);
-        Node<K,V> [] oldTable = table;
-        table = newTable;
         size = 0;
+        Node<K,V> [] oldTable = table;
+        table = new Node[table.length * INCREASE_ARRAY_SIZE_INDEX];;
+        threshold = (int) (table.length * LOAD_FACTOR);
         for (int i = 0; i < oldTable.length; i++) {
             if (oldTable[i] != null) {
```

Comment:
let's remove this checkling

Reply:
i thougth i need checking to iterate only for exist node. done